Mage/src/main/java/mage/abilities/common/PutIntoGraveFromBattlefieldAllTriggeredAbility.java

```diff
@@ -63,4 +63,11 @@ public boolean checkTrigger(GameEvent event, Game game) {
     public PutIntoGraveFromBattlefieldAllTriggeredAbility copy() {
         return new PutIntoGraveFromBattlefieldAllTriggeredAbility(this);
     }
+
+    // Not sure if needed?
+    //
+    //@Override
```

Comment:
I'll try to find a card with the trigger that can come back with Serra Paragon. My current test with Aetherwork Marvel does work on master code.

Reply:
You can try to return false in `getShortLivingLKI` to see broken tests and find cards/effects that can potentially reproduce bad use cases for such triggers.